main.py

```diff
@@ -34,11 +35,16 @@ def consumer_scheduler(loop, source_queue, db_queue):
 def main():
     # Initialize Yara Processing queue
     main_loop = asyncio.get_event_loop()
-    main_loop.run_until_complete(CONFIG.init_db())
-    main_loop.run_until_complete(CONFIG.init_redis_pool())
-    APP_LOGGER.info("Initialized Schema")
 
-    APP_LOGGER.info("Logging up and running")
+    pg_pool = PgPool()
+    main_loop.run_until_complete(pg_pool.init_db())
+
+    if CONFIG.REDIS_CONFIG:
+        redis_pool = RedisConnectionPool()
+        main_loop.run_until_complete(redis_pool.init_redis_pool())
+    else:
+        APP_LOGGER.warning("No Redis Connection Configured falling back to simple Asyncio Queues")
```

Comment:
Probably log this in `info` level. This doesn't seem like a warning if the user actually wants to run it this way